### ISSUE TITLE
Can you setup using command line

### ISSUE THREAD

**Issue by 45bcdf** [Audio4Linux/JDSP4Linux#79]
I have a linux machine that I am using to capture audio from my turntable via Audacity's "monitor audio" function and then run convolution in JDSP4Linux for room correction. It is a pretty old computer that I am trying to run this on and as a result, my current process is too much for the CPU to handle, load averages go above 100%, which results in dropouts and poor audio. If I ran this system with without a DE I would likely be able to keep everything under 100% CPU usage which would hopefully solve my current issue. 

TL;DR - can I set up JDSP4Linux using the command line via SSH? 
Any ideas would be much appreciated. I know I could simply go out and get a better/more powerful computer but that really isn't in the cards at the moment. Cheers

**Comment by ThePBone** [comment]
I have yet to try a headless setup, but two methods should work.
This app was built with Qt, so you can use their platform plugins to change how the app is drawn to the screen. By default, it aborts when no X server, framebuffer, etc. are present.

### Method 1: Disable GUI
```
jamesdsp -platform offscreen --watch
```
The `--watch` parameter keeps track of changes to the effect config file at `~/.config/jamesdsp/audio.conf`. You can use a text editor to edit it, and when you save, the changes get automatically applied (you could also use scripts to automate this since there is no GUI). 

### Method 2: Use GUI remotely using a built-in VNC server
Qt provides a VNC platform plugin that works standalone (no X server or DE required). 
```
jamesdsp -platform vnc
```
You can then connect from your other computer to the VNC server at port 5900 and configure JamesDSP remotely via its GUI.
____
Apart from that, there isn't any additional setup required in most cases. The output device should get chosen automatically; however, I'm not sure how this works when no DE is present.

You could try to manually specify an output device in that case. If necessary, create/update the config file at `~/.config/jamesdsp/application.conf` with these properties:
```
AudioOutputDevice=bluez_output.64_03_7F_2E_2B_3A.1
AudioOutputUseDefault=false
```
Replace the value of `AudioOutputDevice` with the node name of your speakers. Use `pw-dump | grep \"node.name\"` to get a list.
